Libraries/Components/Touchable/Touchable.js

```diff
@@ -712,12 +714,21 @@ var TouchableMixin = {
 
       var shouldInvokePress =  !IsLongPressingIn[curState] || pressIsLongButStillCallOnPress;
       if (shouldInvokePress && this.touchableHandlePress) {
+        this._playClickSound();
```

Comment:
Should probably add the platform check to make sure it doesn't do unnecessary stuff on ios.